boxsdk/object/file.py

```diff
@@ -340,3 +340,47 @@ def add_comment(self, message):
             object_id=response['id'],
             response_object=response,
         )
+
+    def get_watermark(self):
```

Comment:
We should be consistent between this and something like `collaborate` with regards to the presence of the methods on `web_link` objects.  Any thoughts on which you would prefer: either having methods exist that don't work on certain object types, or duplicating code across multiple leaf types?